studio/config/docker/Dockerfile.dev

```diff
@@ -30,4 +39,19 @@ RUN pip3 install --no-cache-dir --upgrade pip && \
     poetry config virtualenvs.create false
 RUN poetry install --no-root --with dev
 
+# Copy application code
```

Comment:
Dockerfile.dev is not supposed to be used for ECS.
Please let me know why you need to edit this file. (Let's have a conversation)